analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Invocation.cs

```diff
@@ -823,6 +829,18 @@ public static class Debug
             new SETestContext(code, AnalyzerLanguage.CSharp, Array.Empty<SymbolicCheck>()).Validator.ValidateTagOrder("End");
         }
 
+        [DataTestMethod]
+        [DataRow("int?")]
+        [DataRow("bool?")]
+        public void Invocation_DebugAssert_NullableHasValue_Simple(string argType) =>
+            DebugAssertValues("arg.HasValue", argType).Should().SatisfyRespectively(x => x.Should().HaveOnlyConstraint(ObjectConstraint.NotNull));
```

Comment:
NIT: `arg.GetValueOrDefault()` might be interesting to see as well.

Reply:
I don't understand this case. `GetValueOrDefault()` returns `int`, for example. That we can test, but we can't learn anything from it.